sphinx_jupyterbook_latex/transforms.py

```diff
@@ -30,6 +30,37 @@ def replace_node_cls(
     src_node.replace_self([node])
 
 
+def findall(node: nodes.Element):
```

Comment:
@AakashGfude does this require the use of `docutils>=0.18`

Reply:
@mmcky no, this function will return node.traverse for cases where findall is not present. i.e, lower versions.